README.md

```diff
@@ -162,7 +160,7 @@ Feel free to check the [issues page](https://github.com/brhanuhailu/vet_clinic/i
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank Microverse
+I would like to thank Microverse and everyone who has contributed to this project
```

Comment:
- [ ] In this milestone, you are expected to create a database according to the image below 👇🏾 

   ![image](https://github.com/brhanuhailu/vet_clinic/assets/84156714/4ac58da8-0c51-4d49-8810-180d37af7e32)

Kindly create a file named - `schema_based_on_diagram.sql`, and use `CREATE TABLE` command to create the various tables, then use foreign and primary keys to implement the associations between them.

Reply:
@bobb-Rob  Thank you very much done